src/resources/views/corporation/mining/ledger.blade.php

```diff
@@ -50,6 +51,9 @@
                                 <span rel="id-to-name">{{ $character_id }}</span>
                             </a>
                         </td>
+                        <td>
+                            <div class="character-id-to-main-character">{{ $character_id }}</div>
```

Comment:
Can you apply the same logic from https://github.com/eveseat/web/pull/194 here ?